documentation.html

```diff
@@ -392,12 +409,56 @@ <h4 class="display-inlineBlock">Toggle Button as Radio Button</h4>
               </div>
             </div>
           </section>
-          
+
+        </section>
+
+
+
+        <!-- Dark button class starts here -->
+        <div>
+          <section class="instruction-block" id="dark-mode">
```

Comment:
You put the `section` element inside a `div` element but it shouldn't be inside a div. remove the parent div.